central/node/datastore/store/postgres/store.go

```diff
@@ -67,8 +66,9 @@ type Store interface {
 	Delete(ctx context.Context, id string) error
 	GetIDs(ctx context.Context) ([]string, error)
 	GetMany(ctx context.Context, ids []string) ([]*storage.Node, []int, error)
-	// GetNodeMetadata gets the node without scan/component data.
+	// GetNodeMetadata and GetManyNodeMetadata returns the node without scan/component data.
```

Comment:
nit:  my preference would be to put the comment on each function instead of grouping them.  But I understand why you are doing it this way.  So I leave it to you.